### ISSUE TITLE
Date format not being applied to following pages in Firefox

### ISSUE THREAD

**Issue by ghost** [aledeg/FreshRSS-extensions#10]
Hi, I modified your JS code as shown in these issue: https://github.com/FreshRSS/Extensions/issues/109
The new format is applied to the first page loaded, but for the following the default format is applied. Tested on chrome and it works fine.
Is this an issue related to my code or a Firefox issue? Is there anything that can be done?
Thanks!

**Comment by aledeg** [comment]
@maguilara I've checked the extension code to see if I can reproduce your behavior. I couldn't. The dates are modified as expected even after the first page is loaded.
Then I've suspected something in your code but again the dates are modified as well. So I cannot reproduce the behavior you've mentioned at all.

For information, I am currently using Firefox 84.0.2 on archlinux With a lot of extensions. What is your configuration?
